sharepoint/sharepoint-ps/sharepoint-online/New-SPOContainerType.md

```diff
@@ -0,0 +1,214 @@
+
+You must be a SharePoint Embedded Administrator to run this cmdlet.
+
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+New-SPOContainerType -ContainerTypeName ContosoLegal -OwningApplicationId a735e4af-b86e-0000-93ba-1faded6c39e1 -AzureSubscriptionId 564e9025-f7f5-xxx9-9ddd-4cdxxxx1755 -ResourceGroup prod-resources -Region EastUS
+```
+Example output of the New-SPOContainerType cmdlet
+```powershell
+Are you sure you want to perform this action?
+Performing the operation "New-SPOContainerType".
+[Y] Yes [A] Yes to All [N] No [L] No to All [S] Suspend [?] Help (default is "Y"): Y
+ContainerTypeId     : 4f0af585-8dcc-0000-223d-661eb2c604e4
+ContainerTypeName   : ContosoLegal
+OwningApplicationId : a735e4af-b86e-0000-93ba-1faded6c39e1
+AzureSubscriptionId : 564e9025-f7f5-xxx9-9ddd-4cdxxxx1755
+ResourceGroup       : prod-resources
+Region              : EastUS
+Classification      : Standard
+CreationDate        : 7/2/2024
+ExpiryDate          : 
+```
+
+### Example 2  
+```powershell
+New-SPOContainerType –TrialContainerType - ContosoLegal -OwningApplicationId a735e4af-b86e-0000-93ba-1faded6c39e1
+```
+
+Example output of the New-SPOContainerType cmdlet
+
+```powershell
+Are you sure you want to perform this action?
+Performing the operation "New-SPOContainerType".
+[Y] Yes [A] Yes to All [N] No [L] No to All [S] Suspend [?] Help (default is "Y"): Y
+ContainerTypeId                              ContainerTypeName          OwningApplicationId               Classification
+------------------------------------        --------------------   -------------------------------        -----------------
+4f0af585-8dcc-0000-223d-661eb2c604e4            ContosoLegal       a735e4af-b86e-0000-93ba-1faded6c39e1         Trial
+```
+
+
+
+## PARAMETERS
+
+### -ContainerTypeName
+
+This parameter names your ContainerType for your SharePoint Embedded Application
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OwningApplicationId
+
+This parameter specifies the ID of the SharePoint Embedded application.  
+
+```yaml
+Type: String
+Parameter Sets: 
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AzureSubscriptionId
+
+This parameter describes the Azure subscription ID to which the container type needs to be associated.
+
+```yaml
+Type: String
+Parameter Sets: 
+Aliases:
+Applicable: SharePoint Online
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+### -ResourceGroup
+
+THis parameter describes the resource group to be used for the associated container type
```

Comment:
>type [](http://example.com/codeflow?start=84&length=4)

(here and other places and other files)

make sure your sentences end with a `.`